applications: asset_tracker_v2: Disable LwM2M bootstrap option (#8385)
nrfconnect/sdk-nrf: applications/asset_tracker_v2/doc/cloud_module.rst

Configurations for LwM2M integration layer
------------------------------------------

When building for LwM2M, the cloud module's default configuration is to communicate with AVSystem's `Coiote Device Management`_, with a runtime provisioned `Pre-shared key (PSK)`_ set by the :kconfig:option:`CONFIG_LWM2M_INTEGRATION_PSK` option.

> This enables the device to work with `Coiote Device Management`_ without provisioning the PSK to the modem prior to running the application.

Reviewer: ```suggestion
This enables the device to work with `Coiote Device Management`_ without provisioning the PSK to the modem before running the application.
```

To allow the device to communicate with other LwM2M servers, modify the default configuration by changing the following Kconfig options:

* :kconfig:option:`CONFIG_LWM2M_CLIENT_UTILS_SERVER`
* :kconfig:option:`CONFIG_LWM2M_RD_CLIENT_SUPPORT_BOOTSTRAP`
* :kconfig:option:`CONFIG_LWM2M_INTEGRATION_ENDPOINT_PREFIX`
* :kconfig:option:`CONFIG_LWM2M_INTEGRATION_PSK`
* :kconfig:option:`CONFIG_LWM2M_INTEGRATION_PROVISION_CREDENTIALS`

> See :ref:`server setup <server_setup_lwm2m>` for information on how the `Coiote Device Management server`_ can be configured to communicate with the application using the default PSK.

Reviewer: ```suggestion
See :ref:`server setup <server_setup_lwm2m>` for information on how you can configure the `Coiote Device Management server`_  to communicate with the application using the default PSK.
```

.. important::

>    In production it is not recommended to use the default PSK that is automatically provisioned by the application.

Reviewer: ```suggestion
   In production, it is not recommended to use the default PSK that is automatically provisioned by the application.
```

   If possible, bootstrapping should be enabled to periodically change the PSK used in the connection.

>    The PSK should also be provisioned to the modem prior to running the application.

Reviewer: ```suggestion
   The PSK should also be provisioned to the modem before running the application.
```

>    This can be done by disabling the :kconfig:option:`CONFIG_LWM2M_INTEGRATION_PROVISION_CREDENTIALS` option and provisioning the PSK to a sec tag set by :kconfig:option:`CONFIG_LWM2M_CLIENT_UTILS_SERVER_TLS_TAG` or :kconfig:option:`CONFIG_LWM2M_CLIENT_UTILS_BOOTSTRAP_TLS_TAG`.

Reviewer: ```suggestion
   Disable the :kconfig:option:`CONFIG_LWM2M_INTEGRATION_PROVISION_CREDENTIALS` option and provision the PSK to a sec tag set by :kconfig:option:`CONFIG_LWM2M_CLIENT_UTILS_SERVER_TLS_TAG` or :kconfig:option:`CONFIG_LWM2M_CLIENT_UTILS_BOOTSTRAP_TLS_TAG`.
```
